lib/realtime_web/channels/realtime_channel.ex

```diff
@@ -388,7 +388,7 @@ defmodule RealtimeWeb.RealtimeChannel do
   def limit_channels(%{assigns: %{tenant: tenant, limits: limits}, transport_pid: pid}) do
     key = Tenants.channels_per_client_key(tenant)
 
-    if Registry.count_match(Realtime.Registry, key, pid) > limits.max_channels_per_client do
+    if Registry.count_match(Realtime.Registry, key, pid) > String.to_integer(System.get_env("MAX_CHANNELS_PER_CLIENT") || Integer.to_string(limits.max_channels_per_client)) do
```

Comment:
this first checks (for existing tenants) if `MAX_CHANNELS_PER_CLIENT` and uses that value (or defaults back to the limit existing on the tenant) for backwards compatibility. Both are treated as strings to allow comparison, before finally being converted back to an integer for use/processing.

Reply:
this kind of logic would probably be better to be in `runtime.exs` and then we could just do a `Application.get_env(:realtime, :max_channels_per_client)` 🤔

Reply:
I ended up using `Application.get_env(:realtime, :max_channels_per_client, 100)` just to make sure that the value was always an integer, can remove if you prefer